### ISSUE TITLE
is there a pre-trained model?

### ISSUE THREAD

**Issue by hafidh561** [mr7495/image-classification-spatial#1]
I have some datasets from the internet with tag forest. But not all in there is a forest image and I want to filter it with your model. Can I use your pre-trained model? @mr7495

**Comment by mr7495** [comment]
Hi, yes you can use the trained models on the Intel image classification dataset which includes the forest category. But notice that our goal was to show that training a model using our method achieves higher accuracy than the usual ways, but you can use newer and better models using our method to reach higher classification accuracy.
This link may work for you: https://drive.google.com/file/d/12sfVqbRu39NCUjTcrrSaIjCe9XAJkNtI/view?usp=sharing

**Comment by hafidh561** [comment]
But, in your repository, there is no model used with an exception model with an intel dataset. Or maybe can you convert it into onnx?

**Comment by mr7495** [comment]
I just shared a model with you in the last massage. The models are based on Keras (.hdf5 format). You can use the public modules to convert it to onnx.

**Comment by hafidh561** [comment]
ah, I see because I mostly used PyTorch btw and I am never used TensorFlow before haha. Thanks, bro
